packages/tooltip/src/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,186 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Tooltip hides on ESC 1`] = `
```

Comment:
One thing you can do if you end up keeping these snapshot files is add a label to them:

```js
expect(baseElement).toMatchSnapshot('after focus');
```
```
exports[`Tooltip hides on ESC: after focus 1`] =
```